dll/win32/shimgvw/lang/cs-CZ.rc

```diff
@@ -1,6 +1,6 @@
 /* Translators:
  * 2008-06-24, Radek Liska aka Black_Fox (radekliska at gmail dot com)
- * 2021-02-15, Václav Zouzalík aka Venca24 (vaclav.zouzalik at seznam dot cz)
+ * 2024-05-02, Václav Zouzalík aka Venca24 (vaclav.zouzalik at seznam dot cz)
  */
```

Comment:
I'd recommend updating the copyright headers to our current standards, see for example: https://github.com/reactos/reactos/blob/master/dll/win32/shimgvw/lang/zh-TW.rc

```rc
/*
 * PROJECT:     ReactOS Picture and Fax Viewer
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0+)
 * PURPOSE:     Czech resource file
 * TRANSLATORS: Copyright 2008 Radek Liška (Black_Fox) <radekliska@gmail.com>
 *              Copyright 2021-2024 Václav Zouzalík (Venca24) <vaclav.zouzalik@seznam.cz>
 */
```